src/posit/connect/client.py

```diff
@@ -1,15 +1,24 @@
 from __future__ import annotations
 
+import os
 from requests import Response, Session
 from typing import Optional
 
 from . import hooks, urls
 
 from .auth import Auth
 from .config import Config
+from .oauth import OAuthIntegration
 from .users import Users
 
 
+# Connect sets the value of the environment variable RSTUDIO_PRODUCT = CONNECT
+# when content is running on a Connect server. Use this var to determine if the
+# client SDK was initialized from a piece of content running on a Connect server.
+def is_local() -> bool:
+    return not os.getenv("RSTUDIO_PRODUCT") == "CONNECT"
```

Comment:
It looks like this is only being used [src/posit/connect/external/databricks.py](https://github.com/posit-dev/posit-sdk-py/pull/52/files#diff-b826271d8297d4f972ef39df49328e78f4a7fada12ebe598b587cf42160eaae8). 

Should it be moved there?

This could also go in `Config` if there is reuse. But I'd rename it to something like `run_mode`, `runtime_environment`, etc.

Reply:
👍🏻 Place it where it's used for now.